persistence-api/src/main/java/io/stargate/db/metrics/CDCMetrics.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.stargate.db.metrics;
+
+import static java.util.concurrent.TimeUnit.NANOSECONDS;
+
+import com.codahale.metrics.Counter;
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+
+public final class CDCMetrics {
+  public static final CDCMetrics instance = new CDCMetrics();
+  private static final String METRIC_NAME = "cdc-producer";
+
+  private boolean initialized = false;
+  private MetricRegistry metricRegistry;
+
+  private Counter producerMessagesInFlight;
+  private Meter producerFailures;
+  private Meter producerTimedOut;
+  private Timer producerLatency;
+
+  private CDCMetrics() {}
+
```

Comment:
I think it will be useful to have low granularity metrics for types of `MutationEvents`:
- `deletion-event-count`
- `update-event-count`
etc.
and `schema-changed-count` when there `createTableSchemaAsync` called